go.mod

```diff
@@ -35,6 +35,7 @@ require (
 	github.com/golang/protobuf v1.4.3
 	github.com/golang/snappy v0.0.4
 	github.com/google/gofuzz v1.1.1-0.20200604201612-c04b05f3adfa
+	github.com/google/pprof v0.0.0-20220829040838-70bd9ae97f40
```

Comment:
Can we remove the change of `go.mod` and `go.sum`, seems this PR is not related with dependency.

Reply:
Sorry. I missed  that you used google profile, it is ok with me then.